pom.xml

```diff
@@ -32,6 +32,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot projects, it's usually better to opt for starters rather than individual dependencies (look for spring-boot-starter-validation)